modules/manually-restoring-cluster-etcd-backup.adoc

```diff
@@ -0,0 +1,312 @@
+// Module included in the following assemblies:
+//
+// * disaster_recovery/scenario-2-manually-restoring-cluster-etcd-backup.adoc
+// * post_installation_configuration/cluster-tasks.adoc
+
+
+:_content-type: PROCEDURE
+[id="manually-restoring-cluster-etcd-backup_{context}"]
+= Restoring a cluster manually from an etcd backup
+
+You can use a saved etcd backup to restore a previous cluster state or restore a cluster that has lost the majority of control plane hosts. The procedure described below only uses the already existing control plane nodes.
+
+[NOTE]
+====
+The standard restore procedure requires the complete recreation of 2 control plane nodes, which might be complex for clusters installed with UPI. 
+====
+
+[NOTE]
+====
+If your cluster uses a control plane machine set, see "Troubleshooting the control plane machine set" for a more simple etcd recovery procedure.
+====
+
+[IMPORTANT]
+====
+When you restore your cluster, you must use an etcd backup that was taken from the same z-stream release. For example, an {product-title} 4.7.2 cluster must use an etcd backup that was taken from 4.7.2.
+====
+
+.Prerequisites
+
+* Access to the cluster as a user with the `cluster-admin` role.
+* A healthy control plane host to use as the recovery host.
+* SSH access to *all* control plane hosts, with the ability to switch to `root` user.
+* A backup directory containing both the etcd snapshot and the resources for the static pods, which were from the same backup. The file names in the directory must be in the following formats: `snapshot_<datetimestamp>.db` and `static_kuberesources_<datetimestamp>.tar.gz`.
+
+[IMPORTANT]
+====
+For non-recovery control plane nodes, it is not required to establish SSH connectivity or to stop the static pods. You can delete and recreate other non-recovery, control plane machines, one by one.
+====
+
+.Procedure
+
+. Select a control plane host to use as the recovery host. This is the host that you will run the restore operation on.
```

Comment:
```suggestion
. Select a control plane host to use as the recovery host. This is the host where you will run the the etcd database regeneration operation on.
```

Reply:
The restore is applied in _all_ nodes, therefore the original wording might be misleading.

Reply:
How about, "Select a control plane host to use as the recovery host. This is the host where you run the etcd database regeneration operation on."?

Reply:
Actually, after reviewing https://access.redhat.com/articles/6965021, I think there is no "main" recovery host _at all_.
The _Regenerate the etcd database from the backup_ step in the KCS is the same on all 3 control plane nodes.

Reply:
I'd say that we might completely remove the mention of a "recovery host".
In this procedure, all nodes are equal (only at the end, when we restart the etcd static pods, we have a etcd leader)

Reply:
@acancell-redhat Instead of "recovery host" should I just mention "hosts"?